app/controllers/pages/secondary_skip_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Pages::SecondarySkipController < PagesController
+  before_action :ensure_branch_routing_feature_enabled
+
+  def new
+    secondary_skip_input = Pages::SecondarySkipInput.new(form: current_form, page:)
+    render locals: {
+      secondary_skip_input:,
+      back_link_url: show_routes_path(form_id: current_form.id, page_id: page.id),
+    }
+  end
+
+  def create
+    secondary_skip_input = Pages::SecondarySkipInput.new(secondary_skip_input_params)
+
+    if secondary_skip_input.submit
+      redirect_to show_routes_path(form_id: current_form.id, page_id: page.id)
+    else
+      render template: "pages/secondary_skip/new", locals: {
+        secondary_skip_input:,
+        back_link_url: show_routes_path(form_id: current_form.id, page_id: page.id),
```

Comment:
I think back links can now be directly in the views and we don't need to pass locals at all. Secondary_skip_input can be an instance variable instead.